packages/tools/client-debugger/client-debugger-chrome-extension/src/react-components/ContainerSummaryView.tsx

```diff
@@ -93,13 +93,42 @@ export function ContainerSummaryView(props: ContainerStateViewProps): React.Reac
 		return <Waiting label="Waiting for Container Summary data." />;
 	}
 
-	// TODO: connect/disconnect, close handlers
+	function tryConnect(): void {
+		messageRelay?.postMessage({
```

Comment:
`messageRelay` should never be undefined here, so we should be able to remove the `?`s from our message posting.

Reply:
Removed